python/peacock/ExodusViewer/ExodusViewer.py

```diff
@@ -23,9 +23,9 @@ class ExodusViewer(peacock.base.ViewerBase):
     def commandLineArgs(parser):
         parser.add_argument('--exodus', '-r', nargs='*', default=[], help="A list of ExodusII files to open.")
 
-    def __init__(self, plugins=[VTKWindowPlugin, FilePlugin, GoldDiffPlugin, VariablePlugin, \
+    def __init__(self, plugins=[FilePlugin, GoldDiffPlugin, VariablePlugin, \
                                 MeshPlugin, BackgroundPlugin, ClipPlugin, ContourPlugin, CameraPlugin, \
-                                OutputPlugin, MediaControlPlugin,
+                                OutputPlugin, VTKWindowPlugin, MediaControlPlugin,
```

Comment:
I have no idea why this change is necessary but it is. With the original order the scroll area gets all screwed up when scrolling.